Documentation/Signoffs/PowerSwitch.md

```diff
@@ -0,0 +1,41 @@
+Power Switch
```

Comment:
The switch in isolation is not a subsystem. This needs to consider the power system at large.